api/v1beta1/foundationdbcluster_types.go

```diff
@@ -2609,3 +2609,29 @@ func (clusterStatus *FoundationDBClusterStatus) AddStorageServerPerDisk(serversP
 
 	clusterStatus.StorageServersPerDisk = append(clusterStatus.StorageServersPerDisk, serversPerDisk)
 }
+
+// None represents an empty struct used for creating set where we don't care about the value.
+type None struct{}
+
+// GetCoordinatorSet returns the current coordinator addresses as a set based on the information
+// in the connection string of the FoundationDBClusterStatus
+func (cluster FoundationDBCluster) GetCoordinatorSet() map[string]None {
+	coordinators := make(map[string]None)
+	// Split the connection string to get only the addresses after the @
+	conSplit := strings.Split(cluster.Status.ConnectionString, "@")
```

Comment:
Rather than relying on parsing the connection string, it might be cleaner to get it from the database status.

Reply:
We could also look at the process roles so we can check this by instance ID instead of using the address as an indirection. That may be more future-proof.

Reply:
That sounds reasonable, my idea was to prevent to have to many status calls against the FDB cluster but I guess that's a valid case.